Orange/widgets/unsupervised/tests/test_owhierarchicalclustering.py

```diff
@@ -28,13 +28,12 @@ def _select_data(self):
         return [14, 15, 32, 33]
 
     def _compare_selected_annotated_domains(self, selected, annotated):
-        self.assertTrue(all((var in annotated.domain.variables
-                             for var in selected.domain.variables)))
+        self.assertEqual(annotated.domain.variables,
+                         selected.domain.variables)
         self.assertNotIn("Other", selected.domain.metas[0].values)
         self.assertIn("Other", annotated.domain.metas[0].values)
-        self.assertTrue(
-            all((var in [var.name for var in annotated.domain.metas]
-                 for var in [var.name for var in selected.domain.metas])))
+        self.assertLess(set(var.name for var in selected.domain.metas),
+                        set(var.name for var in annotated.domain.metas))
```

Comment:
No need to change anything here, but: you can also use set comprehensions in such cases. (I keep forgetting this, too.)